src/python_testing/TC_DRLK_2_13.py

```diff
@@ -0,0 +1,656 @@
+                    signingKey=self.signingKey,
+                    verificationKey=self.verificationKey,
+                    groupIdentifier=self.groupIdentifier,
+                    groupResolvingKey=self.groupResolvingKey),
+                    endpoint=1,
+                    timedRequestTimeoutMs=1000)
+                asserts.assert_equal(expected_status, Status.Success)
+        except InteractionModelError as e:
+            logging.exception(f"Got exception when performing SetAliroReaderConfig {e}")
+            asserts.assert_equal(e.status, expected_status, f"Unexpected error returned: {e}")
+
+    async def get_credentials_status(self, credentialIndex: int, credentialType: cluster.Enums.CredentialTypeEnum,
+                                     step, userIndex, credential_exists=True):
+        if step is not None:
+            self.step(step)
+        try:
+
+            if self.pics_guard(self.check_pics("DRLK.S.F0d") and self.check_pics("DRLK.S.C24.Rsp") and self.check_pics(
+                    "DRLK.S.C25.Tx")):
+                credentials_struct = cluster.Structs.CredentialStruct(credentialIndex=credentialIndex,
+                                                                      credentialType=credentialType)
+                response = await self.send_single_cmd(endpoint=1, timedRequestTimeoutMs=1000,
+                                                      cmd=cluster.Commands.GetCredentialStatus(
+                                                          credential=credentials_struct))
+                asserts.assert_true(type_matches(response, Clusters.DoorLock.Commands.GetCredentialStatusResponse),
+                                    "Unexpected return type for GetCredentialStatus")
+                asserts.assert_true(response.credentialExists == credential_exists,
+                                    "Error when executing GetCredentialStatus command, credentialExists={}".format(
+                                        str(response.credentialExists)))
+                asserts.assert_equal(userIndex, response.userIndex,
+                                     f"User Index is not matching, UserIndex={response.userIndex}")
+                return response
+        except InteractionModelError as e:
+            logging.error(e)
+            asserts.assert_equal(e.status, Status.Success, f"Unexpected error returned: {e}")
+
+    async def set_credential_cmd(self, credential_enum: Clusters.DoorLock.Enums.CredentialTypeEnum, credentialIndex,
+                                 operationType, userIndex, credentialData, userStatus, userType, step=None):
+        if step is not None:
+            self.step(step)
+        credentials = cluster.Structs.CredentialStruct(
+            credentialType=credential_enum,
+            credentialIndex=credentialIndex)
+        if self.pics_guard(self.check_pics("DRLK.S.F0d") and self.check_pics("DRLK.S.C22.Rsp")):
```

Comment:
prefer to `if not ...: return` to reduce indent.

Reply:
can we leave it untouched so that code has readability just like test plan spec?